### PR TITLE
GH-66 : Prepare plugin for the release

### PR DESCRIPTION
**Pull request by shreyasikhar**
This PR brings the changes like:
- Update screenshots and gifs with current implementation
- Updates to plugin's `readme.txt` and github's `README.md`
- Updates on the plugin header information
- Updates on `.po` file,`package.json` and `package-lock.json` with a version bump

Note: New plugin version will be 1.1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,7 +34,11 @@ Below is the list of custom fields added by this plugin.
 - Shortcode with caption
 - Custom HTML with tinyMCE editor
 
-**Note:** This plugin works with default themes. In order to use it with custom themes, please add the necessary styling.
+**Notes:**
+
+- This plugin works with default themes. In order to use it with custom themes, please add the necessary styling from your theme to overwrite the plugin CSS.
```
Comment (gagan0123) [inline README.md:39]:
```suggestion
- This plugin works with default themes. In order to use it with custom themes, please add the necessary styling in your theme/child theme to override the plugin's CSS.
```

File: README.md [hunk 2]
```diff
@@ -34,7 +34,11 @@ Below is the list of custom fields added by this plugin.
...
+- The custom fields from this plugin should be used in the submenu for well-structured and responsive UI.
+- Currently, this plugin is not supported on FSE themes.
```
Comment (gagan0123) [inline README.md:41]:
```suggestion
- Currently, this plugin is not supported with FSE themes.
```

File: readme.txt [hunk 3]
```diff
@@ -20,7 +20,11 @@ Below is the list of custom fields this plugin adds.
 - Shortcode with caption
 - Custom HTML with TinyMCE editor
 
-**Note:** This plugin works with default themes. In order to use it with custom themes, please add the necessary styling.
+**Notes:**
+
+- This plugin works with default themes. In order to use it with custom themes, please add the necessary styling from your theme to overwrite the plugin CSS.
```
Comment (gagan0123) [inline readme.txt:25]:
```suggestion
- This plugin works with default themes. In order to use it with custom themes, please add the necessary styling in your theme/child theme to override the plugin's CSS.
```

File: readme.txt [hunk 4]
```diff
@@ -20,7 +20,11 @@ Below is the list of custom fields this plugin adds.
...
+- The custom fields from this plugin should be used in the submenu for well-structured and responsive UI.
+- Currently, this plugin is not supported on FSE themes.
```
Comment (gagan0123) [inline readme.txt:27]:
```suggestion
- Currently, this plugin is not supported with FSE themes.
```

File: readme.txt [hunk 5]
```diff
@@ -192,6 +202,12 @@ Same [GPL](http://www.gnu.org/licenses/gpl-2.0.txt) that WordPress uses!
 
 == Changelog ==
 
+= 1.0.3 =
+FIXED
+ * Fix UI bugs on default themes
```
Comment (gagan0123) [inline readme.txt:207]:
```suggestion
 * Fix UI bugs with default themes
```

**Review by gagan0123 — CHANGES_REQUESTED** [review]

**Comment by gagan0123** [comment]
@shreyasikhar 
Nice catch of version number in `package.json` and copyright year, I missed it in the documentation 👍

**Comment by shreyasikhar** [comment]
> @shreyasikhar Nice catch of version number in `package.json` and copyright year, I missed it in the documentation 👍

I will update this in the documentation.

**Review by gagan0123 — APPROVED** [review]

**Review by alvitazwar — APPROVED** [review]
LGTM

**Review by gagan0123 — APPROVED** [review]

**Review by alvitazwar — DISMISSED** [review]
LGTM

**Review by alvitazwar — DISMISSED** [review]

**Review by gagan0123 — APPROVED** [review]

**Pull request merged**